sbin/common/config_init.sh

```diff
@@ -83,6 +83,8 @@ OPENJDK_BUILD_NUMBER
 OPENJDK_CORE_VERSION
 OPENJDK_FEATURE_NUMBER
 OPENJDK_FOREST_NAME
+OPENJDK_FOREST_DIR
+OPENJDK_FOREST_DIR_ABSPATH
```

Comment:
Do we need both these? Just have OPENJDK_FOREST_DIR the defaults to "", to mean not specified

Reply:
No, you are right, and one should be enough. To split the flag and value seemed  to be much more readable to me. Do you insists?

Reply:
Agree with @sxa on the rename here

Reply:
done!